src/js/smooch.jsx

```diff
@@ -212,24 +219,41 @@ export class Smooch {
     }
 
     destroy() {
+        let {embedded} = store.getState().appState;
         disconnectFaye();
         store.dispatch(reset());
+        if (process.env.NODE_ENV !== 'test') {
+            unmountComponentAtNode(this._container);
+        }
+
+        if (embedded) {
+            // retain the embed mode
+            store.dispatch(AppStateActions.setEmbedded(true));
```

Comment:
In the non-embedded mode, the container is created by us and needs to be removed from the DOM. In the embedded version, the container is provided and we don't have control over it.

The real "remove from DOM" action happens couple of line above with `unmountComponentAtNode(this._container);` which remove the actual widget from the DOM and initiate the unmount flow for all components in it.